package.json

```diff
@@ -57,6 +63,7 @@
     "crypto-browserify": "^3.12.0",
     "ethers": "^5.6.9",
     "https-browserify": "^1.0.0",
+    "jest-localstorage-mock": "^2.4.21",
```

Comment:
why you need this dependency? You can just use `window.localStorage.setItem` for it